recipes/jieba3k/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "jieba3k" %}
+{% set version = "0.35.1" %}
+{% set file_ext = "zip" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "980a4f2636b778d312518066be90c7697d410dd5a472385f5afced71a2db1c10" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: 'python setup.py install '
+
+requirements:
+  build:
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - jieba
+
+about:
+  home: http://github.com/fxsjy
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: Chinese Words Segementation Utilities
+  description: " "Jieba" (Chinese for "to stutter") Chinese text segmentation: built to be the best Python Chinese word segmentation module."
+  dev_url: 'http://github.com/fxsjy'
+
+extra:
+  recipe-maintainers: 
+  - ganeshhubale
```

Comment:
I think you need to indent this, which may be the source of the parsing error of the `meta.yaml`

Reply:
I updated meta.yaml as per your suggestions. Please review.
Thank you,